packages/cli/src/services/password.utility.ts

```diff
@@ -9,7 +9,10 @@ export class PasswordUtility {
 		return await hash(plaintext, SALT_ROUNDS);
 	}
 
-	async compare(plaintext: string, hashed: string) {
+	async compare(plaintext: string, hashed: string | null) {
+		if (hashed === null) {
+			return false;
+		}
```

Comment:
Can we use `?? ''` at the callsites so that we keep the signature simple here? This'd avoid having to use `!` also.

Reply:
I would prefer changing the interface here, otherwise we need to keep handling the null case for all callsites and these need to know how to do that. If the function accepts both, the handling of null values can be centralized.